web/src/main/java/com/google/accompanist/web/WebView.kt

```diff
@@ -68,6 +70,7 @@ fun WebView(
     captureBackPresses: Boolean = true,
     navigator: WebViewNavigator = rememberWebViewNavigator(),
     onCreated: (WebView) -> Unit = {},
+    onDispose: (WebView) -> Unit = { it.destroy() },
```

Comment:
I don't mind exposing the callback but I think `it.destroy()` should be moved to onDispose and always called. Is there a valid usecase for not calling it?

Reply:
I was thinking about configuration changes. But it probably requires more work to handle it properly (cf https://github.com/google/accompanist/issues/1178).